Dockerfile

```diff
@@ -6,7 +6,6 @@ ENV DATABASE_URL=postgresql://root@localhost/vmdb_production?encoding=utf8&pool=
 
 RUN yum -y install --setopt=tsflags=nodocs \
                    memcached               \
-                   postgresql-server       \
```

Comment:
I think this is still needed ... right?

Reply:
@carbonin this was related to the change in one of the manageiq-pods Dockerfiles (https://github.com/ManageIQ/manageiq-pods/blob/master/images/manageiq-base/Dockerfile) where I built postgresSQL from source, making this redundant. I will re-visit this change after I have addressed the review comments for that PR, thank you for your feedback.

Reply:
Now that we are following the same method for installing postgreSQL (using RPM) on ppc64le, this will be required, I will revert this change, still working with the Travis team for the service start issue though..